### PR TITLE
Add styling for dialog elements

### PR DESCRIPTION
**Pull request by pietvanzoen**

### CODE REVIEW

File: simple.css [hunk 1]
```diff
@@ -625,6 +627,25 @@ progress:indeterminate::-moz-progress-bar {
   background-color: var(--accent-bg);
 }
 
+
```
Comment (lkhrs) [inline simple.css:630]: Let's remove the blank lines on line 630 and 648.

**Review by lkhrs — CHANGES_REQUESTED** [review]
Thanks for working on this. I noted a minor formatting issue.

**Comment by waldyrious** [comment]
For comparison:

| Before | After |
| ------ | ----- |
| Screen Shot 2023-04-06 at 19 12 15 | Screen Shot 2023-04-06 at 19 13 40 |

(Are the Netlify preview links configurable? I wonder if there's a way to have an additional link that would essentially render the new `index.html` with the old `simple.css`, to allow for convenient before/after comparison like the above.)

**Comment by lkhrs** [comment]
> (Are the Netlify preview links configurable? I wonder if there's a way to have an additional link that would essentially render the new `index.html` with the old `simple.css`, to allow for convenient before/after comparison like the above, which I had to set up by hand.)

The latest commit to `main` is viewable on the CDN subdomain: https://cdn.simplecss.org

**Comment by pietvanzoen** [comment]
@lkhrs No problem and thanks for the prompt review. Removed the whitespace.

**Review by lkhrs — APPROVED** [review]

**Comment by waldyrious** [comment]
> The latest commit to `main` is viewable on the CDN subdomain: [cdn.simplecss.org](https://cdn.simplecss.org/)

Thanks for that info, I wasn't aware of it. Still, while that may be sufficient in many cases, it doesn't include HTML markup that is introduced/edited in a PR. In this case, for example, there would be no dialog element in the `main`-based demo, so a before/after comparison would not be possible without extra work.

**Pull request merged**